fenix/app/src/main/res/values/strings.xml

```diff
@@ -2270,7 +2272,9 @@
     <!-- Clickable text from the contextual onboarding card that links to review quality check support article. -->
     <string name="review_quality_check_contextual_onboarding_learn_more_link">Learn more</string>
     <!-- Caption text to be displayed in review quality check contextual onboarding card above the opt-in button. First parameter is the Fakespot product name. Following parameters are for clickable texts defined in review_quality_check_contextual_onboarding_privacy_policy and review_quality_check_contextual_onboarding_terms_use. In the phrase "Fakespot by Mozilla", "by" can be localized. Does not need to stay by. -->
-    <string name="review_quality_check_contextual_onboarding_caption">By selecting “Yes, try it” you agree to %1$s by Mozilla’s %2$s and %3$s.</string>
+    <string name="review_quality_check_contextual_onboarding_caption" moz:RemovedIn="121" tools:ignore="UnusedResources">By selecting “Yes, try it” you agree to %1$s by Mozilla’s %2$s and %3$s.</string>
+    <!-- Caption text to be displayed in review quality check contextual onboarding card above the opt-in button. First parameter is the Fakespot product name. Following parameters are for clickable texts defined in review_quality_check_contextual_onboarding_privacy_policy and review_quality_check_contextual_onboarding_terms_use. -->
+    <string name="review_quality_check_contextual_onboarding_caption_2">By selecting “Yes, try it” you agree to %1$s’s %2$s and %3$s.</string>
```

Comment:
One thing about this string: Content Design has decided to update the string on both iOS and Android to [this](https://github.com/mozilla-mobile/firefox-ios/pull/17131/files#diff-b743e29e051f39265ce2645f435dc519504660e0aa4e4ba5bad75b26b0e4f400R3902
) string.
Should we already update it here as well, to avoid further changes soon?
CC @klowermoz

Reply:
Nice catch. I am landing the changes with the onboarding link fies over in https://github.com/mozilla-mobile/firefox-android/pull/4324

Reply:
@dsmithpadilla, let's land this uplift and then we can land the uplift for the other string changes + link text a11y fix in https://github.com/mozilla-mobile/firefox-android/pull/4339

Reply:
will do thank you both!